### DISCUSSION TITLE
정오표 - [p. 348: 실습코드 6-2] 관련 문의

### DISCUSSION THREAD

**Discussion by gunuduru** [fsoftwareengineer/todo-application#43] [질문과 답변] [answered]
안녕하세요, 오늘 acornpub 사이트에서 정오표를 확인하다가 궁금한것이 생겨 질문드립니다.

[p. 348: 실습코드 6-2]
pip install awsebcli --upgrade -user
->
pip install awsebcli --upgrade –user

위의 수정은 - 가 – 로 바뀐것....뿐인 것 같은데...

어떤 차이인지 궁금합니다.

감사합니다.

**Comment by fsoftwareengineer** [comment, marked as answer]
대시가 두개입니다 --user
특정 브라우저나 문서에서 --가 이어져 보이는 것 같습니다. 출판사측에 요청하여 수정하도록 하겠습니다.
감사합니다.

**Reply by gunuduru** [reply]
감사합니다!
